docs/configurations.md

```diff
@@ -1,53 +1,70 @@
 # ScalarDB Configurations
 
-This document describes the configurations for ScalarDB.
+This page describes the available configurations for ScalarDB.
 
-## Transaction manager configurations
+## Transaction managers
 
-ScalarDB has several transaction manager implementations, such as Consensus Commit, gRPC, and JDBC, and you can specify one of the implementations with the `scalar.db.transaction_manager` property.
-This section describes the configurations for each transaction manager.
+Implemented within ScalarDB are two transaction managers: Consensus Commit and gRPC. You can specify one of the transaction managers by using the `scalar.db.transaction_manager` property and then specify the configurations for the transaction manager.
```

Comment:
`gRPC` (ScalarDB Server) is deprecated, so it should be `ScalarDB Cluster`.
Although we set `cluster` in the `transaction_manager` setting, `ScalarDB Cluster` is not a transaction manager, so this expression may be a bit misleading.

Reply:
@feeblefakie Understood. For branches `3.9` and later, I think we agreed on changing mentions of ScalarDB Server to ScalarDB Cluster. 

Since this PR is for branches `3.8` and earlier, I think this sentence is OK, right?

Reply:
Oh, it is for `3.8` and earlier. Then, it shouldn't be merged into master?

Reply:
As far as I know, we should be able to still merge this PR into `master` and then update `master` again in the PR that addresses `3.10` and `3.9` with the ScalarDB Cluster content. 

That's how I imagined these docs would need to be merged into the branch versions after I submitted this PR and found that ScalarDB Server would need to be replaced with ScalarDB Cluster for `3.10` and `3.9`.